src/Pages/yechanhong/Main/Main.js

```diff
@@ -2,6 +2,35 @@ import React, { Component } from "react";
 import "./Main.scss";
 
 export class MainHong extends Component {
+  constructor() {
+    super();
+    this.state = {
+      commentValue: "",
+      commentBtnVaildation: false,
+    };
+  }
+
+  handleCommentValue = (e) => {
+    this.setState(
+      {
+        commentValue: e.target.value,
+      },
+      () => this.handleCommentButton()
+    );
+  };
+
+  handleCommentButton = () => {
+    if (this.state.commentValue.length >= 1) {
+      this.setState({
+        commentBtnVaildation: true,
+      });
+    } else {
+      this.setState({
+        commentBtnVaildation: false,
+      });
+    }
+  };
```

Comment:
- 객체 단축 속성명을 사용하면 더 간단하게 할 수 있겠지만 우선 아래 코드처럼 수정해볼 수 있겠네요! Boolean 값을 잘 활용하는 훈련 많이 해보시면 좋습니다 :)

```suggestion
  handleCommentButton = () => {
    const btnValidation = this.state.commentValue.length >= 1;
    this.setState({ commentBtnVaildation : btnValidation})
    }
  };
```

Reply:
객체 단축 속성명을 통해서 간단하게 구현했습니다!